provisioning/standalone/vscode/golang/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM harbor.crownlabs.polito.it/crownlabs-standalone/vscode-base:latest
+
+ENV SUDO_FORCE_REMOVE yes
+RUN apt-get update && \
+    apt-get install -y gcc wget && \
+    apt-get purge -y sudo
+
+ADD https://go.dev/dl/go1.17.6.linux-amd64.tar.gz /usr/local/go.tar.gz
+ADD https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh install.sh
```

Comment:
Can be installed with `go install` IIRC

Reply:
![image](https://user-images.githubusercontent.com/33266330/162176295-447b19e2-a35e-46f0-9dd2-a25a70e7284d.png)
From the golangci-lint documentation https://golangci-lint.run/usage/install/

Reply:
Ok, I think most of the mentioned issues wouldn't apply to this use case, but your approach is nonetheless fine.